Automattic/wc-calypso-bridge
# Adds setup checklist base

This PR adds a new base page for the setup checklist (see #64). It adds a 'Setup' page to the top of the WooCommerce navigation. It also hides all notices on the page, so we can have a nice clean task list.

There is not much to see in this PR, but this figured like a good cut-off point for review before actually building the task-list code.

<img width="1250" alt="screen shot 2018-10-26 at 10 45 12 am" src="https://user-images.githubusercontent.com/689165/47573847-47aaa200-d90c-11e8-9d0f-0024c823a1ee.png">

To Test:
* Verify you see the 'Setup' link under WooCommerce, above orders
* Verify the page loads with the correct navigation, and is totally blank (no notifications)
* Set `atomic-ecommerce-setup-checklist-complete` to true, and the page should go away.
